### ISSUE TITLE
BA.2 sublineage with S:I68T and C7471T circulating in UK (568 seqs globally)

### ISSUE THREAD

**Issue by FedeGueli** [cov-lineages/pango-designation#566]
Follow up from #524 
S:I68T was spotted by @olias120676 in several european clades
Here i want to propose the one carrying S:I68T and then  C7471T
It reached 376 sequences mainly in England and Wales.
https://cov-spectrum.org/explore/World/AllSamples/Past6M/variants?aaMutations=S%3AI68T&nucMutations=C7471T&pangoLineage=BA.2*&aaMutations1=S%3AI68T&nucMutations1=C7471T&

Usher [tree](https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_a566_d99900.json?branchLabel=aa%20mutations&c=gt-nuc_7471&label=nuc%20mutations:C241T,T670G,C2790T,C3037T,G4184A,C4321T,C9344T,A9424G,C9534T,C9866T,C10029T,C10198T,G10447A,C10449A,C12880T,C14408T,C15714T,C17410T,A18163G,C19955T,A20055G,C21618T,T21765C,T22200G,G22578A,C22674T,T22679C,C22686T,A22688G,G22775A,A22786C,C22792T,G22813T,T22882G,G22992A,C22995A,A23013C,A23040G,A23055G,A23063T,T23075C,A23403G,C23525T,T23599G,C23604A,C23854A,G23948T,A24424T,T24469A,C25000T,C25584T,C26060T,C26270T,C26577G,G26709A,C26858T,A27259C,G27382C,A27383T,T27384C,C27807T,A28271T,C28311T,G28881A,G28882A,G28883C,A29510C)  is clear:
![chrome_screenshot_1650302047679](https://user-images.githubusercontent.com/87669813/163845831-887c9c37-6489-4c27-833f-6bfb666984b0.png)

Growth advantage over BA.2 is moderate around 22% in UK
![chrome_screenshot_1650302595787](https://user-images.githubusercontent.com/87669813/163847116-ecc6417e-0f69-41cc-8d6b-7fe30d55a960.png)

**Comment by c19850727** [comment]
Hi @FedeGueli , here's a cluster of potential BA.1*+BA.2* recombinants (5 seqs, with breakpoint at NSP3) that interestingly have a conserved **Spike I68T** substitution ((and another unique N:Q7K substitution):

EPI_ISL_11384598, 11679909, 11814006, 11938911, 12070160

![image](https://user-images.githubusercontent.com/86835836/163898326-cff217e9-2753-4098-82c1-d7854af2eafa.png)

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_10615_bde6b0.json?c=userOrOld&label=nuc%20mutations:T21765C,C28292A

**Comment by corneliusroemer** [comment]
Thanks for the proposal.

If S:68T is potentially homoplasic. So Usher can't necessarily be trusted on the sequence of mutations. It's hard to tell whether C7471T came first or second. Both is possible.

Albeit I slightly favour C7471T first because  that mutation was present in the UK before.

**Comment by corneliusroemer** [comment]
This keeps growing in the UK, I'd be in favour of a designation of C7471T + S:I68T.

**Comment by FedeGueli** [comment]
List of sequences:
[contributors (5).csv](https://github.com/cov-lineages/pango-designation/files/8638218/contributors.5.csv)

This sublineage goes on growing.
@chrisruis @corneliusroemer @AngieHinrichs

**Comment by corneliusroemer** [comment]
This is about a quarter of the UK BA.2 + S:68T. One needs to decided whether there'll be one big BA.2 + S:68T lineage, or whether we designate many BA.2 + S:68T + defining-distinction.

This Belgian #565 makes up about half of all global S:68T, the rest remains undesignated. I've proposed two other regionally distinctive ones in #609 #610

**Comment by FedeGueli** [comment]
568 seqs today

**Comment by AngieHinrichs** [comment]
FWIW in the current UCSC/UShER tree this is a branch of BA.2.36 (BA.2 + C22792T + T21765C/S:I68T) although as Cornelius says either C7471T or T21765C/S:I68T could have come first.  

But there's plenty of evidence for C7471T having homoplasies of its own.  There's a branch with 282 sequences with C7471T and no change on S:I68 (BA.2 + C25416T > G26577C > C7471T, starting with MMR/DSMRC-095/2022|ON038768.1|2022-02-19) but those are mostly from Myanmar/Thailand/Japan/Australia.  There are a dozen French sequences (e.g. France/ARA-HCL022054062502/2022) with BA.2 T9866C > G29332T > C7471T, 15 sequences from Denmark (e.g. Denmark/DCGC-417690/2022) with BA.2 + C22792T > C25624T > C14925T > C1594T,C12043T > C7471T.  And there are 86 mostly USA sequences (e.g. USA/NJ-DHSS-B1181997/2022) with BA.2 + C22792T > C25624T > A24433G > C22450T > C7471A,A18010G.  (BA.2 + C22792T > C25624T = BA.2.9)  To find sequences from England with 7471T and not S:I68T I had to do a cov-spectrum query, and they're scattered all over BA.2 (in branches with fewer than 10 descendants which is the threshold I used in taxonium's mutation search), with lots of other mutations.

So no harm in calling this a sublineage of BA.2.36, I think.
